lib/travis/api/v3/renderer/settings.rb

```diff
@@ -0,0 +1,18 @@
+module Travis::API::V3
+  module Renderer::Settings
+    extend self
+
+    AVAILABLE_ATTRIBUTES = [:settings]
+
+    def available_attributes
+      AVAILABLE_ATTRIBUTES 
+    end
+
+    def render(settings, **)
+      {
+        :@type => 'settings'.freeze,
+        :settings => settings.to_h
```

Comment:
Maybe these should be at top level rather than nested in settings?

Otherwise embedding them in a repository, if we were to offer that feature, would look like this:

``` json
{
  "@type": "repsitory",
  "slug": "example/example",
  "settings": {
    "@type": "settings",
    "settings": { }
  }
}
```

ie, the actual settings would be accessible as `repo.settings.settings`.  Or was that intentional in order to allow adding attributes to the settings object that aren't actual settings?